Sources/NIO/EventLoop.swift

```diff
@@ -660,6 +662,9 @@ final public class MultiThreadedEventLoopGroup: EventLoopGroup {
                 do {
                     /* we try! this as this must work (just setting up kqueue/epoll) or else there's not much we can do here */
                     let l = try! SelectableEventLoop(thread: t)
+                    
+                    threadSpecificEventLoop.currentValue = l
```

Comment:
So the core problem here right now is that this leaks each event loop. The TSV does not (and indeed cannot) free the data stored in it when the thread exits, so this will keep the loop in memory forever.

When the loop is cleaned up, we need to nil out this TSV.

Reply:
Ah I thought it will... that’s easy to fix. Let me do it later